### DISCUSSION TITLE
Restore Torso T1 to the compatible MIDI devices over USB

### DISCUSSION THREAD

**Discussion by brentmoreland** [SynthstromAudible/DelugeFirmware#2938] [Ideas]
I would like to use the Torso T1 to create patterns in the Deluge.  The only way I can do it is to use the TRS MIDOUT on the Torso to the  the DIN IN port on the deluge, using a special TRS to DIN cable.  I have to power the torso, of course.  What I want to be able to do is connect the two units with USB.  I have successfully connected a QuNexus midi controller, and it works over USB.  I read someone's comment in June 2023 that said it was working for him with Deluge3.x firmware, and it stopped working with 4.0 and beyond.  I'm using Chopin 1.3 beta community firmware.  It would be awesome if this compatibility could be restored.  I have tried learning the old way, and with the MIDI Follow method.  Both of these methods work with the QuNexus, btw.

**Comment by stellar-aria** [comment]
We're currently working on a completely new USB backend, so the situation will likely change and be very different from the current one after that's integrated. If things aren't working after that, we can re-examine the problem at that time.

**Comment by brentmoreland** [comment]
Thanks for the quick response. I have to say, this dev team is exceptional in its dedication. I’m on my fourth deluge (I sold three already. Just got my 4th one yesterday.)I will NOT sell the fourth one. lol. I’m happy to contribute my tiny voice is the bus picture of things. On Nov 19, 2024, at 7:20 PM, Katherine Whitlock ***@***.***> wrote:﻿
We're currently working on a completely new USB backend, so the situation will likely change and be very different from the current one after that's integrated. If things aren't working after that, we can re-examine the problem at that time.
